packages/sample-app/sample_app/strands_example.py

```
@@ -0,0 +1,410 @@
+import os
+import json
+import subprocess
```

Ruff (0.12.2)

`subprocess` imported but unused

Remove unused import: `subprocess`

(F401)